test/alternator/test_table.py

```diff
@@ -253,13 +253,13 @@ def test_table_streams_off(dynamodb):
     # Unfortunately, boto3 doesn't allow us to pass StreamSpecification=None.
     # This is what we had in issue #5796.
 
-@pytest.mark.xfail(reason="streams not yet implemented")
 def test_table_streams_on(dynamodb):
-    table = create_test_table(dynamodb,
-        StreamSpecification={'StreamEnabled': True, 'StreamViewType': 'OLD_IMAGE'},
-        KeySchema=[{ 'AttributeName': 'p', 'KeyType': 'HASH' }],
-        AttributeDefinitions=[{ 'AttributeName': 'p', 'AttributeType': 'S' }]);
-    table.delete();
+    for type in [ 'OLD_IMAGE', 'NEW_IMAGE', 'KEYS_ONLY', 'NEW_AND_OLD_IMAGES']:
```

Comment:
Creating all these tables and then doing nothing with them is really slow. I suggest you put those tables to good use after creating them. In other words, instead of this test (which you can remove) create in test_streams.py four different fixtures - one creating a table with an OLD_IMAGE, one with a NEW_IMAGE, etc., and create four tests like this one which check that each of these tables works. You can then use these four fixtures in other tests - listing them, checking (eventually) the records actually appear, and so on.

Reply:
Sure. It somewhat does against the concept of small unit tests though. Maybe fixture permutations?
Or simply push the type checking to an eventual test that also tests the data coming out.